examples/nodejs-nestjs/README.md

```diff
@@ -0,0 +1,43 @@
+<a href="https://arcjet.com" target="_arcjet-home">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://arcjet.com/arcjet-logo-minimal-dark-mark-all.svg">
+    <img src="https://arcjet.com/arcjet-logo-minimal-light-mark-all.svg" alt="Arcjet Logo" height="128" width="auto">
+  </picture>
+</a>
+
+# Arcjet Protection with NestJS for Node.js
+
+This example shows how to use Arcjet to protect [NestJS](https://nestjs.com/) apps using the Node.js SDK.
+
```

Comment:
As this is a bit different from how our other examples work with the rules all defined in a single place on the instantiated client, I think it's worth adding some more explanation. Perhaps point out where our SDK is integrated and how to add more rules.